src/js/themes/grommet.js

```diff
@@ -8,7 +8,8 @@ export const grommet = deepFreeze({
       background: '#ffffff',
     },
     font: {
-      family: '"San Francisco", "Helvetica Neue", Helvetica, Arial, sans-serif',
+      family:
+        '-apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu, Cantarell, "Fira Sans", "Droid Sans",  "Helvetica Neue", Arial, sans-serif,  "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";',
```

Comment:
Can you articulate where this list came from? These are not the fonts @L0ZZI wanted to promote for the `grommet` theme.

Reply:
All the fonts are contained on there it is a super set. This lists comes from github and atlassian. If we want to do system fonts this is the way to go

Reply:
Let's discuss with @L0ZZI tomorrow then.

Reply:
I do like this stack. It's a little more inclusive to make a better display on typography respective systems. I'm in support of it. Any cost to the change?

Reply:
no it is a super set. we should be fine.